src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -426,9 +426,9 @@ private void ResetForNewDocument()
             foreach (var ws in Workspaces.OfType<HomeWorkspaceModel>())
             {
                 foreach (var node in ws.Nodes)
-                    node.ForceReExecuteOfNode = true;
+                    node.MarkNodeAsModified();
```

Comment:
I think resetting for a new Revit document we need all nodes to be force re-executed, no? In which case we need `MarkNodeAsModified(forceExecute: true)`. I'm just being afraid that removing that will cause the nodes to not re-execute.